index.html

```diff
@@ -401,11 +401,11 @@ <h2>Glossary</h2>
 
 <p><dfn class="lint-ignore export">Standalone vowel</dfn>. Vowel sounds that are not immediately preceded by a consonant sound; they typically appear at the beginning or in the middle of a word. In Brahmi-derived scripts standalone vowels are often represented using [=independent vowel=] letters.</p>
 
-<p><dfn class="lint-ignore export" data-lt="supplementary code point">Supplementary characters</dfn>. Beyond the [=Basic Multilingual Plane=] the Unicode character set also contains space for around a million additional code point positions. Characters in this latter range are referred to as supplementary characters. In the UTF-16 encoding, supplementary characters are encoded using  pairs of surrogate characters.</p>
+<p><dfn class="lint-ignore export" data-lt="supplementary code point">Supplementary character</dfn>. Beyond the [=Basic Multilingual Plane=] the Unicode character set also contains space for around a million additional code point positions. Characters in this latter range are referred to as supplementary characters. In the UTF-16 encoding, each supplementary character is encoded using a pair of [=surrogates=].</p>
 
-<p><dfn class="lint-ignore" data-lt="surrogate">Surrogate code point</dfn>. <a href="https://www.unicode.org/glossary/#surrogate_code_point">Unicode definition</a>: <q>A Unicode code point in the range U+D800..U+DFFF. Reserved for use by UTF-16, where a pair of surrogate code units (a high surrogate followed by a low surrogate) “stand in” for a [=supplementary code point=].</q> This term is also defined by [[INFRA]].</p>
+<p><dfn class="lint-ignore" data-lt="surrogate">Surrogate code point</dfn>. <a href="https://www.unicode.org/glossary/#surrogate_code_point">Unicode definition</a>: <q>A Unicode code point in the range <code>U+D800..U+DFFF</code>. Reserved for use by UTF-16, where a pair of surrogate code units (a high surrogate followed by a low surrogate) “stand in” for a [=supplementary code point=].</q> This term is also defined by [[INFRA]].</p>
```

Comment:
If we just use `<code>` (without `class="kw" translate="no"`), we can use the backtick (`) syntax.

Reply:
I'd prefer to use HTML in most cases rather than mixing in more markdown. However, your callout is the correct one: I added class=kw and translate=no.